app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
I'd prefer if the new feature was in its own subnet-specific controller so as not to remove the use of helpers.